dtfs-central-api/src/cron-scheduler-jobs/delete-acbs-durable-function-logs/index.ts

```diff
@@ -10,9 +9,16 @@ const { ACBS_DURABLE_FUNCTIONS_LOG_DELETION_SCHEDULE } = process.env;
  * @throws {WriteConcernError} If the deletion fails to write
  */
 const deleteCompleteAcbsDurableFunctionLogs = async (): Promise<void> => {
-  const { acknowledged } = await deleteAllCompleteAcbsDurableFunctionLogs(generateSystemAuditDetails());
-  if (!acknowledged) {
-    throw new WriteConcernError();
+  try {
+    const { acknowledged } = await deleteAllCompleteAcbsDurableFunctionLogs(generateSystemAuditDetails());
+
+    if (!acknowledged) {
+      throw new WriteConcernError();
+    }
+  } catch (error) {
+    if (!(error instanceof DocumentNotDeletedError)) {
```

Comment:
do we want to log if the error is `DocumentNotDeleted`?

Reply:
This will happen anytime there's no documents to delete so isn't really an error in this case - I'm happy to log but I don't think its necessary